### PR TITLE
[feature][doc] update doc for jdbc-connector

### PR DESCRIPTION
**Pull request by chl-wxp**
###  Purpose of this pull request
update doc for jdbc-connector

###  Does this PR introduce any user-facing change?
no

###  How was this patch tested?
Only change the documentation

### Check list

* [x] add schema_save_mode parameter and introduction
* [x] add data_save_mode parameter and introduction
* [x] add custom_sql parameter and introduction 
* [x] update table parameter

### CODE REVIEW

**Comment by Hisoka-X** [comment]
Please use default template.
image

**Comment by chl-wxp** [comment]
> Please use default template. image

OK

File: docs/en/connector-v2/sink/Jdbc.md [hunk 1]
```diff
@@ -26,30 +26,33 @@ support `Xa transactions`. You can set `is_exactly_once=true` to enable it.
...
+| generate_sink_sql                         | Boolean | No       | false                        |
+| xa_data_source_class_name                 | String  | No       | -                            |
+| max_commit_attempts                       | Int     | No       | 3                            |
+| transaction_timeout_sec                   | Int     | No       | -1                           |
+| auto_commit                               | Boolean | No       | true                         |
+| field_ide                                 | String  | No       | -                            |
+| properties                                | Map     | No       | -                            |
+| common-options                            |         | no       | -                            |
+| schema_save_mode                          | Enum    | no       | CREATE_SCHEMA_WHEN_NOT_EXIST |
+| data_save_mode                            | Enum    | no       | APPEND_DATA                  |
```
Comment (zhilinli123) [inline docs/en/connector-v2/sink/Jdbc.md:54]: Can you do a test Demo for this?
for example : https://github.com/apache/seatunnel/blob/e26006d2aa47df0f92740cf9843dca862db82232/docs/en/connector-v2/source/Jdbc.md?plain=1#L246-L260

File: docs/en/connector-v2/sink/Jdbc.md [hunk 2]
```diff
@@ -231,6 +269,8 @@ sink {
         database = "sink_database"
         table = "sink_table"
         primary_keys = ["key1", "key2", ...]
+        schema_save_mode = "CREATE_SCHEMA_WHEN_NOT_EXIST"
+        data_save_mode="APPEND_DATA"
```
Comment (zhilinli123) [inline docs/en/connector-v2/sink/Jdbc.md:290]: Add some descriptive information to tell the user what he does?

**Review by hailin0 — APPROVED** [review]

**Review by ic4y — APPROVED** [review]

**Pull request merged**